src/components/Inputs/GeojsonPropertyMatch.js

```diff
@@ -0,0 +1,52 @@
+import i18n from '@dhis2/d2-i18n'
+import { ReactFinalForm, CheckboxFieldFF, InputFieldFF } from '@dhis2/ui'
+import React from 'react'
+import { StyledField } from '../index'
+import { OrgUnitIdSchemeCore } from './index'
+
+const { useField, Field } = ReactFinalForm
+
+const MATCH_PROPERTY_NAME = 'matchProperty'
+const MATCH_PROPERTY_LABEL = i18n.t(
+    'Match GeoJSON property to organisation unit scheme'
+)
+const MATCH_PROPERTY_DATATEST = 'input-match-property'
+
+const NAME = 'geojsonProperty'
+const DATATEST = 'input-geojson-property'
+const LABEL = i18n.t('GeoJSON property name')
```

Comment:
These potentially wouldn't get translated at runtime. Do they need to be constants or can all these just be added directly in the jsx?